README.md

```diff
@@ -264,6 +272,9 @@ Further, you can claim back travel expenses for journeys to customer offices thr
 
 Many players in mondora have chosen to publish their salary internally for others to see. As a new employee you can choose if you would also like to publish your salary, and if you wish to see others' salaries. In order to access this information you must join the Slack channel \#salaries. Here you will then find a spreadsheet with salary info and you can add your own if you like.
 
+#### MBO and awards
+The company is a place where each of us takes care of the rest. That's why we love the idea of giving a prize to teams instead of individuals. A shared award is within a timeframe where the team is self-distributing it. If the expiry date of the grant comes, the prize goes in a pool and used for other projects. You are suggested to think in the perspective of a colleague and to try understanding why is essential have that money on her view.
```

Comment:
The company is a place where each of us takes care of the rest. That's why we love the idea of giving a prize to teams instead of individuals. Bonuses are distributed to the whole team who has to share it out in a specific timeframe.  If the team is not able to agree on how to share the bonus in the specified time, the whole sum will be used for other projects instead. The idea is to think about the bonus from the perspective of your colleagues and try to understand why it would be essential for them to receive it.